microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -163,6 +171,60 @@ func reconcileResource(ctx context.Context, dynamicClient *dynamic.DynamicClient
 					}
 				}
 
+				if createdObj.GetKind() == "Deployment" {
```

Comment:
I think put the logic here would apply the config multiple times when iterate the graph, I thought, this would be done after adjustConfigmap in preProcessUserConfigmap ?

Reply:
Hi @KfreeZ `adjustConfigmap` in `preProcessUserConfigmap` occurs before the deployment for K8s resources, but according to our previous, we need to apply after deployment, and there are some overwriting occurring if only change one configmap.